controllers/binding/snapshotenvironmentbinding_adapter_test.go

```diff
@@ -431,57 +431,10 @@ var _ = Describe("Binding Adapter", Ordered, func() {
 		}, time.Second*10).Should(BeTrue())
 	})
 
-	It("ensures ephemeral environment is deleted for the given pipelineRun ", func() {
```

Comment:
Why was this test moved to the end of file?
It generates hard to read git diff, can we keep the test on the same place? I don't see reason for moving it

Reply:
Moved back